### ISSUE TITLE
stop action for DummyShutter causes error

### ISSUE THREAD

**Issue by Flop2006** [pimatic/pimatic#692]
The stop action for the DummyShutter class causes an error:
"Next action of "stop rolladen-schlafzimmer" is ambigous."

Rule:
```if off-button is pressed then stop rollladen-schlafzimmer```
Device:
```{"class": "DummyShutter",
"id": "rollladen-schlafzimmer",
"name": "Rollladen"
}```

**Comment by leader21** [comment]
There's a typo. Either write rolladen-schlafzimmer or rollladen-schlafzimmer

**Comment by Flop2006** [comment]
sorry, didn´, copy paste it. It is "rollladen-schlafzimmer" in both cases.

**Comment by sweetpi** [comment]
@Flop2006 Could you post all your devices (config)?

**Comment by Flop2006** [comment]
```
  "devices": [
    {
      "id": "steckdose1",
      "name": "Steckdose1",
      "class": "HomeduinoRFSwitch",
      "protocols": [
        {
          "name": "switch6",
          "options": {
            "systemcode": 0,
            "programcode": 1
          }
        }
      ]
    },
    {
      "id": "steckdose2",
      "name": "Steckdose2",
      "class": "HomeduinoRFSwitch",
      "protocols": [
        {
          "name": "switch6",
          "options": {
            "systemcode": 0,
            "programcode": 2
          }
        }
      ]
    },
    {
      "id": "steckdose3",
      "name": "Steckdose3",
      "class": "HomeduinoRFSwitch",
      "protocols": [
        {
          "name": "switch6",
          "options": {
            "systemcode": 0,
            "programcode": 3
          }
        }
      ]
    },
    {
      "id": "steckdose4",
      "name": "Steckdose4",
      "class": "HomeduinoRFSwitch",
      "protocols": [
        {
          "name": "switch6",
          "options": {
            "systemcode": 0,
            "programcode": 4
          }
        }
      ]
    },
    {
      "id": "system-time",
      "name": "System Time",
      "class": "ShellSensor",
      "attributeName": "time",
      "attributeType": "string",
      "attributeUnit": "",
      "command": "date +%d.%m.%Y\" \"%H:%M\" \"Uhr\" \"%Z",
      "interval": 60000
    },
    {
      "class": "DummySwitch",
      "id": "urlaubsmodus",
      "name": "Urlaubsmodus",
      "xConfirm": true
    },
    {
      "id": "jemandzuhause",
      "name": "Jemand Zuhause",
      "class": "VariablesDevice",
      "variables": [
        {
          "name": "anybodyathome",
          "expression": "$anybodyathome",
          "type": "string",
          "label": "Jemand Zuhause"
        }
      ]
    },
    {
      "id": "kodi-player",
      "name": "Kodi",
      "class": "KodiPlayer",
      "host": "192.168.2.119",
      "port": 9090
    },
    {
      "class": "DummyShutter",
      "id": "rollladen-schlafzimmer",
      "name": "Rollladen"
    },
    {
      "id": "wetterstation",
      "name": "Wetterstation",
      "class": "HomeduinoRFWeatherStation",
      "protocols": [
        {
          "name": "weather1",
          "options": {
            "id": 20,
            "channel": 1
          }
        }
      ],
      "values": [
        "temperature",
        "humidity"
      ]
    },
    {
      "id": "relais1",
      "class": "GpioSwitch",
      "name": "Relais 1",
      "gpio": 4,
      "inverted": true,
      "defaultState": true
    },
    {
      "id": "relais2",
      "class": "GpioSwitch",
      "name": "Relais 2",
      "gpio": 17,
      "inverted": true,
      "defaultState": true
    },
    {
      "id": "relais3",
      "class": "GpioSwitch",
      "name": "Relais 3",
      "gpio": 22,
      "inverted": true,
      "defaultState": true
    },
    {
      "id": "relais4",
      "class": "GpioSwitch",
      "name": "Relais 4",
      "gpio": 27,
      "inverted": true,
      "defaultState": true
    },
    {
      "id": "led-buttons",
      "name": "LED Fernbedienung",
      "class": "HomeduinoRFButtonsDevice",
      "buttons": [
        {
          "id": "on-button",
          "text": "An/Aus",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "on/off",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "mode+-button",
          "text": "Mode+",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "mode+",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "mode--button",
          "text": "Mode-",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "mode-",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "speed--button",
          "text": "Speed-",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "speed-",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "speed+-button",
          "text": "Speed+",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "speed-",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "color+-button",
          "text": "Farbe+",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "color+",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "color--button",
          "text": "Farbe-",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "color-",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "bright+-button",
          "text": "Heller",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "bright+",
                "id": 12521
              }
            }
          ]
        },
        {
          "id": "bright--button",
          "text": "Dunkler",
          "protocols": [
            {
              "name": "led3",
              "options": {
                "command": "bright-",
                "id": 12521
              }
            }
          ]
        }
      ]
    },
    {
      "id": "radio",
      "name": "Radio",
      "class": "ButtonsDevice",
      "buttons": [
        {
          "id": "ffh",
          "text": "FFH",
          "xConfirm": true
        },
        {
          "id": "hr3",
          "text": "HR3",
          "xConfirm": true
        },
        {
          "id": "rockantenne",
          "text": "ROCKANTENNE",
          "xConfirm": true
        }
      ]
    },
    {
      "id": "restart",
      "name": "Restart",
      "class": "ButtonsDevice",
      "buttons": [
        {
          "id": "pimatic",
          "text": "Pimatic Neustart",
          "confirm": true
        },
        {
          "id": "raspberry",
          "text": "Raspberry Neustart",
          "confirm": true
        }
      ]
    },
    {
      "id": "WOL",
      "name": "WOL Button",
      "class": "ButtonsDevice",
      "buttons": [
        {
          "id": "wolserver",
          "text": "Server"
        },
        {
          "id": "wollaptop",
          "text": "Laptop"
        }
      ]
    },
    {
      "id": "handy1",
      "name": "Handy1",
      "class": "PingPresence",
      "host": "192.168.2.136",
      "interval": 10000
    },
    {
      "id": "handy2",
      "name": "Handy2",
      "class": "PingPresence",
      "host": "192.168.2.115",
      "interval": 10500
    },
    {
      "id": "online_google",
      "name": "Online Status",
      "class": "PingPresence",
      "host": "8.8.8.8",
      "interval": 60000
    },
    {
      "id": "online_server",
      "name": "Server",
      "class": "PingPresence",
      "host": "192.168.2.112",
      "interval": 9900
    },
    {
      "id": "system-uptime",
      "name": "System uptime",
      "class": "ShellSensor",
      "attributeName": "Uptime",
      "attributeType": "string",
      "attributeUnit": "",
      "command": "bash /home/pimatic-app/uptime.sh",
      "interval": 60000
    },
    {
      "class": "SystemSensor",
      "id": "syssensor_cpu",
      "name": "CPU Auslastung",
      "attributes": [
        {
          "name": "cpu"
        }
      ]
    },
    {
      "class": "SystemSensor",
      "id": "syssensor_memory",
      "name": "RAM Auslastung",
      "attributes": [
        {
          "name": "memory"
        }
      ]
    },
    {
      "class": "SystemSensor",
      "id": "syssensor_database_size",
      "name": "Datenbankgroesse",
      "attributes": [
        {
          "name": "dbsize",
          "interval": 360000
        }
      ]
    },
    {
      "class": "SystemSensor",
      "id": "syssensor_diskusage",
      "name": "Festplattenbelegung",
      "attributes": [
        {
          "name": "diskusage",
          "path": "/"
        }
      ]
    },
    {
      "class": "SystemSensor",
      "id": "syssensor_temp",
      "name": "Systemtemperatur",
      "attributes": [
        {
          "name": "temperature",
          "interval": 360000
        }
      ]
    },
    {
      "id": "openweather",
      "class": "OpenWeatherDevice",
      "name": "Wetter heute",
      "location": "Daheim,Deutschland",
      "lang": "de",
      "units": "metric",
      "timeout": 600000,
      "xAttributeOptions": [
        {
          "name": "temperature",
          "displaySparkline": true
        },
        {
          "name": "pressure",
          "hidden": true
        },
        {
          "name": "windspeed",
          "hidden": true
        },
        {
          "name": "rain",
          "hidden": true
        },
        {
          "name": "snow",
          "hidden": true
        }
      ]
    },
    {
      "id": "openforecast",
      "class": "OpenWeatherForecastDevice",
      "name": "Wetter morgen",
      "location": "Daheim,Deutschland",
      "lang": "de",
      "units": "metric",
      "timeout": 600000,
      "day": 1,
      "xAttributeOptions": [
        {
          "name": "pressure",
          "hidden": true
        },
        {
          "name": "windspeed",
          "hidden": true
        },
        {
          "name": "rain",
          "hidden": true
        },
        {
          "name": "snow",
          "hidden": true
        }
      ]
    },
    {
      "id": "sonnenaufgang",
      "name": "Sonnenaufgang",
      "class": "ShellSensor",
      "attributeName": "time",
      "attributeType": "string",
      "attributeUnit": "",
      "command": "curl -s http://weather.yahooapis.com/forecastrss?w=xxxxxxxx | grep astronomy | awk -F\\\" '{print $2}'|{ date -f - +%R; }",
      "interval": 43200000
    },
    {
      "id": "sonnenuntergang",
      "name": "Sonnenuntergang",
      "class": "ShellSensor",
      "attributeName": "time",
      "attributeType": "string",
      "attributeUnit": "",
      "command": "curl -s http://weather.yahooapis.com/forecastrss?w=xxxxxxxx | grep astronomy | awk -F\\\" '{print $4}'|{ date -f - +%R; }",
      "interval": 43200000
    }
  ],
```

**Comment by p4co86** [comment]
same error for me with the dummy shutter device

**Comment by sweetpi** [comment]
Sorry for the long waiting time... fixed in pimatic v0.8.84.
